src/components/slider/HomePageTopSlider.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div>
+        <carousel :per-page="1" :paginationActiveColor="'#00aeb0'" :autoplay="true" :autoplayTimeout="1500" :loop="true">
+            <slide class="slide_img slide_1">
```

Comment:
use v-for instead multiple slide. for class name you can use v-for counter (https://vuejs.org/v2/guide/list.html, second example).
If you don't know how to do slider with v-for ask me.